mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -1077,27 +1126,39 @@ static int64_t retrieveSplitKValueImpl(StringRef perfConfig) {
 }
 
 static int64_t retrieveSplitKValue(rock::GemmFeatures features,
-                                   StringRef perfConfig) {
+                                   StringAttr perfConfig) {
+  auto attnPerfConfig = AttnPerfConfigAttr::get(perfConfig);
+  if (attnPerfConfig)
+    return attnPerfConfig.getSplitKFactor();
+
   if (isAccel(features)) {
     return retrieveSplitKValueImpl<rock::InitParamsAccel>(perfConfig);
   }
   return retrieveSplitKValueImpl<rock::InitParamsNonAccel>(perfConfig);
 }
 
-bool isSplitKRequested(rock::GemmFeatures features, StringRef perfConfig) {
+bool isSplitKRequested(rock::GemmFeatures features, StringAttr perfConfig) {
   return retrieveSplitKValue(features, perfConfig) > 1;
 }
 
 bool isSplitKRequested(ModuleOp mod, StringRef perfConfig) {
+  auto perfConfigAttr = StringAttr::get(mod->getContext(), perfConfig);
```

Comment:
Check if MIXR-CAPI tests need to be updated. 
https://github.com/ROCm/rocMLIR/blob/develop/mlir/test/CAPI/mixr_split_k.cpp